report/prep.R

```diff
@@ -0,0 +1,81 @@
+# Dependencies ------------------------------------------------------------
+
+suppressPackageStartupMessages(library(readr))
+suppressPackageStartupMessages(library(rgdal))
+suppressPackageStartupMessages(library(dplyr))
+
+# Globals -----------------------------------------------------------------
+
+root <- "~/Work/Idalab/intelligent-zoning-engine"
```

Comment:
Die `prep.R` ist eh eigentlich nur für mich, um die  Datenstrukturen vorzubereiten, die dann in den `rmarkdown::render` call (Zeile 75-81) wandern. War ein schneller Weg für mich, dass zu testen, ohne die shiny app zu starten.